### ISSUE TITLE
'No matches for oh' connection failure

### ISSUE THREAD

**Issue by Heidistein** [jgeboski/purple-facebook#196]
Debug log tell me:
```
(13:25:27) util: Writing file accounts.xml to directory /home/aheidinga/.purple
(13:25:27) util: Writing file /home/aheidinga/.purple/accounts.xml
(13:25:27) util: Writing file blist.xml to directory /home/aheidinga/.purple
(13:25:27) util: Writing file /home/aheidinga/.purple/blist.xml
(13:25:30) account: Connecting to account 1Heidistein1.
(13:25:30) connection: Connecting. gc = 0x5628646712d0
(13:25:30) http: Performing new request 0x562864e21800 to b-api.facebook.com.
(13:25:30) dnsquery: Performing DNS lookup for b-api.facebook.com
(13:25:30) dns: Wait for DNS child 28126 failed: No child processes
(13:25:30) dns: Created new DNS child 28134, there are now 1 children.
(13:25:30) dns: Successfully sent DNS request to child 28134
(13:25:30) dns: Got response for 'b-api.facebook.com'
(13:25:30) dnsquery: IP resolved for b-api.facebook.com
(13:25:30) proxy: Attempting connection to 2a03:2880:20:4f25:face:b00c:0:14c9
(13:25:30) proxy: Connecting to b-api.facebook.com:443 with no proxy
(13:25:30) proxy: Connection in progress
(13:25:30) proxy: Connecting to b-api.facebook.com:443.
(13:25:30) proxy: Connected to b-api.facebook.com:443.
(13:25:30) nss: SSL version 3.3 using 128-bit AES-GCM with 128-bit AEAD MAC
Server Auth: 256-bit ECDSA, Key Exchange: 256-bit ECDHE, Compression: NULL
Cipher Suite Name: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
(13:25:30) nss: subject=CN=*.facebook.com,O="Facebook, Inc.",L=Menlo Park,ST=CA,C=US issuer=CN=DigiCert SHA2 High Assurance Server CA,OU=www.digicert.com,O=DigiCert Inc,C=US
(13:25:30) nss: partial certificate chain
(13:25:30) certificate/x509/tls_cached: Starting verify for b-api.facebook.com
(13:25:30) certificate/x509/tls_cached: Checking for cached cert...
(13:25:30) certificate/x509/tls_cached: ...Found cached cert
(13:25:30) nss/x509: Loading certificate from /home/aheidinga/.purple/certificates/x509/tls_peers/b-api.facebook.com
(13:25:30) certificate/x509/tls_cached: Peer cert matched cached
(13:25:30) nss/x509: Exporting certificate to /home/aheidinga/.purple/certificates/x509/tls_peers/b-api.facebook.com
(13:25:30) util: Writing file /home/aheidinga/.purple/certificates/x509/tls_peers/b-api.facebook.com
(13:25:30) nss: Trusting CN=*.facebook.com,O="Facebook, Inc.",L=Menlo Park,ST=CA,C=US
(13:25:30) certificate: Successfully verified certificate for b-api.facebook.com
(13:25:31) skype: Sending: 'SEARCH MISSEDCHATMESSAGES'
(13:25:32) http: Request 0x562864e21800 performed successfully.
(13:25:32) http: Performing new request 0x562864fa2f00 to graph.facebook.com.
(13:25:32) dnsquery: Performing DNS lookup for graph.facebook.com
(13:25:32) dns: Successfully sent DNS request to child 28134
(13:25:32) dns: Got response for 'graph.facebook.com'
(13:25:32) dnsquery: IP resolved for graph.facebook.com
(13:25:32) proxy: Attempting connection to 2a03:2880:20:4f06:face:b00c:0:1
(13:25:32) proxy: Connecting to graph.facebook.com:443 with no proxy
(13:25:32) proxy: Connection in progress
(13:25:32) proxy: Connecting to graph.facebook.com:443.
(13:25:32) proxy: Connected to graph.facebook.com:443.
(13:25:33) nss: SSL version 3.3 using 128-bit AES-GCM with 128-bit AEAD MAC
Server Auth: 256-bit ECDSA, Key Exchange: 256-bit ECDHE, Compression: NULL
Cipher Suite Name: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
(13:25:33) nss: subject=CN=*.facebook.com,O="Facebook, Inc.",L=Menlo Park,ST=CA,C=US issuer=CN=DigiCert SHA2 High Assurance Server CA,OU=www.digicert.com,O=DigiCert Inc,C=US
(13:25:33) nss: partial certificate chain
(13:25:33) certificate/x509/tls_cached: Starting verify for graph.facebook.com
(13:25:33) certificate/x509/tls_cached: Checking for cached cert...
(13:25:33) certificate/x509/tls_cached: ...Found cached cert
(13:25:33) nss/x509: Loading certificate from /home/aheidinga/.purple/certificates/x509/tls_peers/graph.facebook.com
(13:25:33) certificate/x509/tls_cached: Peer cert matched cached
(13:25:33) nss/x509: Exporting certificate to /home/aheidinga/.purple/certificates/x509/tls_peers/graph.facebook.com
(13:25:33) util: Writing file /home/aheidinga/.purple/certificates/x509/tls_peers/graph.facebook.com
(13:25:33) nss: Trusting CN=*.facebook.com,O="Facebook, Inc.",L=Menlo Park,ST=CA,C=US
(13:25:33) certificate: Successfully verified certificate for graph.facebook.com
(13:25:34) http: Request 0x562864fa2f00 performed successfully.
(13:25:34) connection: Connection error on 0x5628646712d0 (reason: 0 description: No matches for oh)
(13:25:34) account: Disconnecting account 1Heidistein1 (0x562864ff9f20)
(13:25:34) connection: Disconnecting connection 0x5628646712d0
(13:25:34) connection: Destroying connection 0x5628646712d0
(13:25:35) util: Writing file accounts.xml to directory /home/aheidinga/.purple
(13:25:35) util: Writing file /home/aheidinga/.purple/accounts.xml
```

**Comment by jgeboski** [comment]
Please send me a proper [debugging log](https://github.com/jgeboski/purple-facebook/wiki/Debugging-Logs).

**Comment by jgeboski** [comment]
This should be resolved as of [25a255f](https://bitbucket.org/pidgin/main/commits/25a255f32eee7218886f930aeb3413469a8d3e0a). A debugging log would have been nice, but I am running short on time to patch bugs.
